Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -49,26 +52,44 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
             // https://colinbarrebrisebois.com/2011/03/07/gdc-2011-approximating-translucency-for-a-fast-cheap-and-convincing-subsurface-scattering-look/
 
             {
-                thickness = max(shadowRatio, surface.transmission.thickness);
+                thickness = max(transmissionDistance, surface.transmission.thickness);
```

Comment:
nit: this might yield visual cutoff between the two curves.  Perhaps introduce smoothing?

Reply:
This would only influence the thick object transmission mode, which might be out of the scope of this PR?  
The current O3DE implementation for thick objects hasn't been changed, only the name of the parameter `shadowRatio` for `transmissionDistance` in this case.  
Still, it could be done!